package.json

```diff
@@ -24,6 +25,7 @@
     "fastify-sqlite": "^1.1.0",
     "geocodio-library-node": "^1.6.1",
     "lodash": "^4.17.21",
+    "pdf-parse": "^1.1.1",
     "qs": "^6.11.2",
```

Comment:
Actually, should these and cheerio be devDependencies instead of actual dependencies? AFAIK they're not needed for the deployed API.

I guess I don't know how GitHub Actions are configured and whether this distinction matters to those.